src/app/components/JumpTo/fixtureData.ts

```diff
@@ -73,6 +73,144 @@ const pidginArticleFixtureWithJumpToBlock = {
                 ],
               },
             },
+            {
+              id: 'e57fb0e0',
+              type: 'relatedContent',
+              model: {
```

Comment:
Could get rid of this now?

Reply:
I have removed this. I have added some article page tests that check the component is present based on the page data instead (data added in the test).